### PR TITLE
More work on the story mode

### PR DESCRIPTION
**Pull request by alexpearce**
See issue #137 for motivation.

This PR begins the reworking of the first lessons, up to and including “Fun with LoKi Functors”. This means changes to the style and tone to make the lessons more approachable and more consistent with each other, and also adding challenges that should be given to the students during the lessons.

I'll push more changes to this branch for the other lessons.

### CODE REVIEW

**Comment by alexpearce** [comment]
One cool this PR also does is add KaTeX. Check out the “Finding data in the Bookkeeping” lesson for an example:

LaTeX up in ur lesson

**Comment by betatim** [comment]
Are we trying to merge this before tomorrow morning?

File: 03-bookkeeping.md [hunk 1]
```diff
@@ -35,38 +46,58 @@ representation of the [event
...
+This sample of simulated events will only contain events where a $D^{*+} \to 
+D^{0}(\to K^{-}\pi^{+})\pi^{+}$ was generated within the LHCb acceptance, 
+although the decay might not have been fully reconstructed. (Not all simulated 
+samples have the same requirements made on the signal decay.)
 
 If you expand the `27163003 (Dst_D0pi,Kpi=DecProdCut)` folder you
-will find a list of different configurations depending on the
-magnet polarity (`MagDown` and `MagUp`) as well as different
-versions of `Pythia` used to generate the events. We will use
+will find a several different subfolders to choose from. The names of these
```
Comment (betatim) [inline 03-bookkeeping.md]: find a several -> find several

File: 05-interactive-dst.md [hunk 2]
```diff
@@ -23,36 +23,19 @@ is not working as expected.
 The file we [downloaded from the grid](04-files-from-grid.html)
 contains simulated data, with stripping and trigger decisions
 and so on. Here we assumed the file you downloaded is called `00035742_00000002_1.allstreams.dst`.
-To take a look at the contents of the TES we need a simple
-options file:
+To take a look at the contents of the TES we to write a small
```
Comment (betatim) [inline 05-interactive-dst.md]: wut?

**Comment by alexpearce** [comment]
Ideally yeah.

File: 05-interactive-dst.md (hunk 2 above)
Comment (alexpearce) [inline 05-interactive-dst.md]: Should have been “To take a look at the contents of the TES, we need to write a small…”. Fixed.

File: 03-bookkeeping.md (hunk 1 above)
Comment (alexpearce) [inline 03-bookkeeping.md]: Fixed.

**Comment by apuignav** [comment]
Great work! I've checked the changes, apart from the comment on the centrally downloaded file I'm happy to merge. I would like another +1, though.

**Comment by betatim** [comment]
I've not had time to read all of it ... if you want to use it for the lesson today coordinate with the other instructor and merge it if they are fine with the changes.

File: 04-files-from-grid.md [hunk 3]
```diff
@@ -44,7 +48,14 @@ dirac-dms-get-file LFN:/lhcb/MC/2012/ALLSTREAMS.DST/00035742/0000/00035742_00000
 
 Again this will take a while but afterwards you should have a file called `00035742_00000002_1.allstreams.dst` in the directory where you called the command.
 
-Since those tuples tend to be quite large, you might want to use your AFS work directory instead of your AFS user directory to store files.
+> ## Downloading the file during a Starterkit lesson {.callout}
+> Lots of people downloading the same file at the same time can be very slow. As a workout, the file is also available on EOS, and can be downloaded to your current directory with the following command:
```
Comment (roelaaij) [inline 04-files-from-grid.md]: workout -> workaround
I do love them workouts :)

**Comment by roelaaij** [comment]
I've had a look and I think this is great. I'm the other instructor this morning and I'd be very happy to use your changes. If there is still time, I'd propose to merge.

File: 04-files-from-grid.md (hunk 3 above)
Comment (alexpearce) [inline 04-files-from-grid.md]: Good spot! I've fixed it.

**Comment by alexpearce** [comment]
Awesome, thanks everyone!

**Pull request merged**